imports/wasi_snapshot_preview1/fs.go

```diff
@@ -1114,18 +1114,15 @@ func fdWriteFn(ctx context.Context, mod api.Module, stack []uint64) (uint32, Err
 
 	var err error
 	var nwritten uint32
+	iovsBuf, ok := mod.Memory().Read(ctx, iovs, iovsCount*8)
```

Comment:
curious if this makes difference :D 
```suggestion
	iovsBuf, ok := mod.Memory().Read(ctx, iovs, iovsCount<<3)
```

Reply:
neat